patches/server/0626-Missing-Entity-API.patch

```diff
@@ -882,6 +904,21 @@ index 5467e4a74b70ff57b49d9e6bc686c493178f8511..56f9630dbe5d18d5ec33dc85f6531723
  
      @Override
      public boolean isRolling() {
+@@ -101,6 +127,14 @@ public class CraftPanda extends CraftAnimals implements Panda {
+         return this.getHandle().getUnhappyCounter();
+     }
+ 
++    // Paper start
++    @Override
++    public boolean isAggressive() {
++        // Pandas are always aggressive if their variant is AGGRESSIVE
++        return (this.getHandle().getEntityData().get(net.minecraft.world.entity.Mob.DATA_MOB_FLAGS_ID) & 4) != 0;
```

Comment:
Why are you overriding this? The nms call overrides the method anyways.

Reply:
Actually, i have read the above comment. I am not sure why tho we still need this. Since this is a deviation of vanilla behavior. I am not sure if we should make this distinction here.

Reply:
It didn't make sense to me as setting aggressive to false and then checking it would still return true. Still, like you said, it also doesn't align with vanilla.
Either way, I'll add a note to the javadoc for this.

Reply:
This behavior is mostly expected id see. Cause even so, it’s just manually editing the flag, and so as long as docs properly say that this may sometimes have undefined behavior in relation to get/set aggressive, I think that’s fine.